config/pipelines.yml

```diff
@@ -237,15 +237,19 @@ test:
         name: *pcr_tag_set_name
         hostname: 'localhost:3000'
       message: *ont_covid_message
-
+        
```

Comment:
This could do with being removed, but isn't actually harming anything. My editor auto-removes this stuff on save.